gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -236,6 +237,11 @@ private static object ToLambda(string stringLambda, string graphName)
             throw new IgnoreException(IgnoreReason.LambdaNotSupported);
         }
 
+        private static object ToT(string enumName, string graphName)
+        {
+            return Enum.Parse(typeof(T), TraversalParser.GetCsharpName(enumName));
```

Comment:
Shouldn't this simply return the string representation of the `enum` for GraphSON 2.0? Stephen mentioned in TINKERPOP-1866:

> When that did happen in GraphSON 2.0, `T` values were simply converted to string.